features/admin.registration-flow-builder.v1/components/element-property-panel/nodes/rules-properties.tsx

```diff
@@ -33,30 +39,51 @@ import "./rules-properties.scss";
 export interface RulesPropertiesPropsInterface extends IdentifiableComponentInterface { }
 
 /**
- * Component to generate the properties for the attribute collector widget.
+ * Rules properties component.
  *
  * @param props - Props injected to the component.
- * @returns The RulesProperties component.
+ * @returns Rules properties component.
  */
 const RulesProperties: FunctionComponent<RulesPropertiesPropsInterface> = ({
     ["data-componentid"]: componentId = "rules-properties-component"
 }: RulesPropertiesPropsInterface): ReactElement => {
 
-    // TODO: Change the glow value
+    // TODO: Change to collect dynamic value from the context
     const {
-        data: RulesMeta
+        data: RuleExpressionsMetaData
     } = useGetRulesMeta("preIssueAccessToken");
 
+    // TODO: Use this function to get the rule value.
+    /* eslint-disable @typescript-eslint/no-unused-vars */
+    const handleGetRuleValue = () => {
```

Comment:
```suggestion
    const handleGetRuleValue = (): void => {
```

Reply:
This is no longer valid with the latest changes.